js/index.js

```diff
@@ -27,3 +27,14 @@ if (hacker1NameLength > hacker2NameLength) {
 // Iteration 3: Loops
 let hacker1InCapital = hacker1.toUpperCase()
 console.log(hacker1InCapital);
+
+const reverseArrayHacker1 = hacker1InCapital.split("").reverse().join("");
+console.log(reverseArrayHacker1);
```

Comment:
great you're using these methods! Just make sure you know exactly how they wokr if you do ;)